modules/templates/helper.go

```diff
@@ -801,6 +806,67 @@ func RenderIssueTitle(ctx context.Context, text, urlPrefix string, metas map[str
 	return template.HTML(renderedText)
 }
 
+// RenderLabel renders a label
+func RenderLabel(label *issues_model.Label) string {
+	labelScope := label.ExclusiveScope()
+
+	textColor := "#111"
+	if label.UseLightTextColor() {
+		textColor = "#eee"
+	}
```

Comment:
Why are we setting this manually, instead of letting the browser what is best?

Reply:
How to let browser decide? There is no JS for such color,  and the HTML are all generated dynamically.

Reply:
Ah, I thought I had already seen that the browser "auto whitens" dark text on background.
But that was probably simply due to my dark reader.